drivers/mct/import_mct.F

```diff
@@ -227,62 +230,66 @@ subroutine import_mct(x2o_o, lsize, perm, jjcpl)
       call fill_global(mval, fval, halo_ps,
      .                 abswnd_da(1-nbdy,1-nbdy,l2ci))
 
-#ifdef PROGCO2
-      if (index_x2o_Sa_co2prog > 0) then
-         n = 0
-         do j = 1, jjcpl
-            do i = 1, ii
-               n = n + 1
-               if     (ip(i,j) == 0) then
-                  atmco2_da(i,j,l2ci) = mval
-               elseif (cplmsk(i,j) == 0) then
-                  atmco2_da(i,j,l2ci) = fval
-               else
-                  ! Atmospheric co2 concentration [ppmv?]
-                  atmco2_da(i,j,l2ci) =
-     .               x2o_o%rAttr(index_x2o_Sa_co2prog,n)
-               endif
+      if (ocn_co2_type == 'prognostic') then
```

Comment:
This would now need a `#ifdef HAMOCC`, too since `ocn_co2_type` will be undefined otherwise

Reply:
Thanks. I will be testing this PR with mct as well.